.changeset/slow-games-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+'@keystone-6/core': major
+---
+
+Changes the return type for `resolveInput` hook for `json` fields.  Previously you used `'DbNull'` or `'JsonNull'` for respective null magic values - you can now use a Javascript `null` value which is mapped by Keystone to a `Prisma.DbNull`.
```

Comment:
```suggestion
Changes the return type for `resolveInput` hook for `json` fields.  Previously you may have used `'DbNull'` or `'JsonNull'` for respective null magic values - you can now always use the Javascript `null` value, as it will always mapped by Keystone to a `Prisma.DbNull`.
```